cmd/run.go

```diff
@@ -876,6 +883,18 @@ func bindRunFlags(cmd *cobra.Command) {
 	util.MustBindPFlag("log.level", cmd.Flags().Lookup("log-level"))
 	util.MustBindEnv("log.level", "LOG_LEVEL")
 
+	cmd.Flags().Bool("trace-enabled", defaultConfig.Trace.Enabled, "enable tracing")
+	util.MustBindPFlag("trace.enabled", cmd.Flags().Lookup("trace-enabled"))
+	util.MustBindEnv("trace.enabled", "OPENFGA_TRACE_ENABLED")
+
+	cmd.Flags().String("trace-grpc-endpoint", defaultConfig.Trace.Endpoint, "the endpoint of the trace collector")
```

Comment:
`trace-otlpgrpc-endpoint` instead? That way it's more clear that it's wiring up the `otlpgrpc` exporter.

Reply:
Used `otlp-grpc` as that seems more common. Done. Thanks!

Reply:
Perfect 👍